bcml/assets/src/js/Profile.jsx

```diff
@@ -53,7 +53,7 @@ class ProfileModal extends React.Component {
                                 variant="primary"
                                 disabled={!this.state.profileName}
                                 onClick={() =>
-                                    this.props.onSave(this.state.profileName, "save")
+                                    this.props.onSave({ name: this.state.profileName }, "save")
```

Comment:
Does this behave as expected? onSave used to take a String, and now it's being given an Object with the String as a property, but I'm not seeing a diff for onSave that changes it to use the Object now

Reply:
This works as expected; all usages of the [handleProfile](https://github.com/NiceneNerd/BCML/blob/4961984a85563ae533c3da2eaba63a4c4fcc3069/bcml/assets/src/js/App.jsx#L268) method (via [onSave](https://github.com/NiceneNerd/BCML/blob/4961984a85563ae533c3da2eaba63a4c4fcc3069/bcml/assets/src/js/Profile.jsx#L56)/[onLoad](https://github.com/NiceneNerd/BCML/blob/4961984a85563ae533c3da2eaba63a4c4fcc3069/bcml/assets/src/js/Profile.jsx#L75)/[onDelete](https://github.com/NiceneNerd/BCML/blob/4961984a85563ae533c3da2eaba63a4c4fcc3069/bcml/assets/src/js/Profile.jsx#L83)) have been standardised to pass the new/existing profile as an object. The relevant methods within the Python API (`_api.py`)have been updated as appropriate, dealing with the profile object as opposed to either the direct profile name, path, etc.

Reply:
Ah, I see where my confusion was. The JS already treated it like an object. Good form.